src/background.js

```diff
@@ -17,13 +17,35 @@ chrome.alarms.onAlarm.addListener(function (alarm) {
           },
           (injectionResults) => {
             for (const frameResult of injectionResults) {
-              if (frameResult.result === "Devin is waiting") {
-                chrome.notifications.create({
-                  type: "basic",
-                  iconUrl: "icon.png",
-                  title: "Devin Status",
-                  message: "Devin is awaiting your response.",
-                });
+              if (frameResult.result.status === "Devin is waiting") {
+                // Retrieve the last notification time for the current tab from storage
+                chrome.storage.local.get(
+                  `lastNotificationTime_${tab.id}`,
+                  function (data) {
+                    const currentTime = new Date().getTime();
+                    // Parse the last message time from the result
+                    const lastMessageTime = Date.parse(
+                      frameResult.result.lastMessageTime,
+                    );
+                    // Check if the current time is different from the last message time for this tab
+                    if (
+                      !data[`lastNotificationTime_${tab.id}`] ||
+                      currentTime - lastMessageTime > 60000
+                    ) {
```

Comment:
Can you determine if the chat history with the recorded time is shown at the bottom, rather than comparing it with the current time?